views.sql

```diff
@@ -113,14 +113,15 @@ CREATE VIEW cyclosm_ways AS
         END AS segregated,
         CASE
             WHEN tags->'oneway:bicycle' IS NOT NULL THEN tags->'oneway:bicycle'
-            WHEN highway='cycleway' AND oneway IS NOT NULL THEN oneway
             WHEN tags->'cycleway' IN ('opposite', 'opposite_lane', 'opposite_track', 'opposite_share_busway')
               OR tags->'cycleway:both' IN ('opposite', 'opposite_lane', 'opposite_track', 'opposite_share_busway')
               OR tags->'cycleway:left' IN ('opposite', 'opposite_lane', 'opposite_track', 'opposite_share_busway')
               OR tags->'cycleway:right' IN ('opposite', 'opposite_lane', 'opposite_track', 'opposite_share_busway')
               OR tags->'cycleway:left:oneway'='-1' OR tags->'cycleway:right:oneway'='-1'
                 THEN 'no'
-            ELSE NULL
+            WHEN oneway IN ('yes', '-1') THEN oneway
```

Comment:
The changes go beyond the description of this MR (defaulting `highway=cycleway` without `oneway` tag to `oneway=no`).

The global setting is changed (`NULL` => `no`) and there is added logic for junctions.

I'm not sure to see why this is needed?

Thanks

Reply:
The idea is that the cycleways/paths are rendered with the correct width when the implied oneway:bicycle=* value is present. This logic simply applies the fallback oneway:bicycle=* value in the view when it is not tagged explicitly.

Reply:
Ok got it, thanks for the fix.